scripts/BeforeBuild.ps1

```diff
@@ -31,8 +31,9 @@ Set-StrictMode -Version Latest
 $ErrorActionPreference = "Stop"
 $InformationPreference = "Continue"
 
-Import-Module $PSScriptRoot\ScriptUtilities.psm1 -Force
-Import-Module $PSScriptRoot\Projects.psm1 -Force
+Import-Module -Force $PSScriptRoot\ScriptUtilities.psm1
+Import-Module -Force $PSScriptRoot\NuGetUtilities.psm1
```

Comment:
>NuGetUtilities [](start = 35, length = 14)

Since we added more NuGet functionality (previously, we had scripts to create packages; now we also have scripts to hide them), I factored the NuGet functionality into its own module.